src/DIRAC/ConfigurationSystem/Client/test/Test_PathFinder.py

```diff
@@ -4,118 +4,151 @@
 from __future__ import division
 from __future__ import print_function
 
-import os
-import unittest
+import pytest
 from diraccfg import CFG
-from DIRAC.ConfigurationSystem.Client.PathFinder import getComponentSection, getServiceFailoverURL, getServiceURL
-from DIRAC.ConfigurationSystem.private.ConfigurationClient import ConfigurationClient
-from DIRAC.ConfigurationSystem.Client.ConfigurationData import gConfigurationData
 
+from DIRAC.ConfigurationSystem.Client import PathFinder
+from DIRAC.ConfigurationSystem.Client.Helpers import Operations
+from DIRAC.ConfigurationSystem.private.ConfigurationData import ConfigurationData
 
-class TestPathFinder( unittest.TestCase ):
-  def setUp( self ):
-    #Creating test configuration file
-    self.testCfgFileName = 'test.cfg'
-    cfgContent='''
-    DIRAC
+localCFGData = ConfigurationData(False)
+mergedCFG = CFG()
+mergedCFG.loadFromBuffer("""
+DIRAC
+{
+  Setup=TestSetup
+  Setups
+  {
+    TestSetup
     {
-      Setup=TestSetup
-      Setups
-      {
-        TestSetup
-        {
-          WorkloadManagement=MyWM
-        }
-      }
+      WorkloadManagement=MyWM
     }
-    Systems
+  }
+}
+Systems
+{
+  WorkloadManagement
+  {
+    MyWM
     {
-      WorkloadManagement
+      URLs
       {
-        MyWM
-        {
-          URLs
-          {
-            Service1 = dips://server1:1234/WorkloadManagement/Service1
-            Service2 = dips://$MAINSERVERS$:5678/WorkloadManagement/Service2
-          }
-          FailoverURLs
-          {
-            Service2 = dips://failover1:5678/WorkloadManagement/Service2
-          }
-        }
+        Service1 = dips://server1:1234/WorkloadManagement/Service1
+        Service2 = dips://$MAINSERVERS$:5678/WorkloadManagement/Service2
       }
-    }
-    Operations{
-      Defaults
+      FailoverURLs
       {
-        MainServers = gw1, gw2
```

Comment:
Why is `MainServers` removed ?

Reply:
In fact, they are in place, just hard to find looking at the changes [L48-50](https://github.com/DIRACGrid/DIRAC/pull/5286/files/1239519f734bc54c4471bb5c14f19361070315d1#diff-2223aee897e51c657236ec817a740552ebf36f7741a9f7417348074d31f93b4bR48-R50)